x-pack/plugins/fleet/server/errors/handlers.ts

```diff
@@ -40,7 +42,12 @@ interface IngestErrorHandlerParams {
 // this type is based on BadRequest values observed while debugging https://github.com/elastic/kibana/issues/75862
 
 const getHTTPResponseCode = (error: IngestManagerError): number => {
-  if (error instanceof RegistryError) {
+  if (error instanceof RegistryResponseError) {
+    // 4xx/5xx's from EPR
+    return 500;
+  }
+  if (error instanceof RegistryConnectionError || error instanceof RegistryError) {
```

Comment:
Adding explicit check on both, even though RegistryConnectionError subclasses from RegistryError. It may help as in-code documentation, and doesn't tie it as much to class-hierarchy.